latest_cov_build.py

```diff
@@ -18,3 +33,51 @@
 with tarfile.open(name, 'r:bz2') as tar:
     tar.extractall()
 os.remove(name)
+
+# Geckodriver download
+# OS information for correct geckodriver version
+name = sys.platform
+bitness = platform.architecture()[0]
+latest_version = get_json('https://api.github.com/repos/mozilla/geckodriver/releases/latest')
+tag_name = latest_version['tag_name']
+base_url = 'https://github.com/mozilla/geckodriver/releases/download/' + tag_name
+
+# Linux OS
+if name == 'linux':
+    if bitness == '64bit':
+        base_url += '/geckodriver-' + tag_name + '-linux64.tar.gz'
+        print(base_url)
+    else:
+        base_url += '/geckodriver-' + tag_name + '-linux32.tar.gz'
+
+# MacOS
+elif name == 'darwin':
+    base_url += '/geckodriver-' + tag_name + '-macos.tar.gz'
+
+# Windows or Cygwin
+elif name == 'cygwin' or name == 'win32':
+    if bitness == '64bit':
+        base_url += '/geckodriver-' + tag_name + '-win64.zip'
+        print(base_url)
+    else:
+        base_url += '/geckodriver-' + tag_name + '-win32.zip'
+
+# Create 'tools/' directory if doesn't exist
+if not os.path.exists('tools'):
+    os.makedirs('tools')
+
```

Comment:
In the previous branches, you could store the file name instead of appending to base_url, then here you could do:
```Python
urlretrieve('{}/{}'.format(base_url, file_name), file_name)

if file_name.endswith('zip'):
    with zipfile...
elif file_name.endswith('tar.gz'):
    with tarfile...
```

This way we can avoid most of the differences between the .zip and the .tar.gz branches.